### PR TITLE
Pages fix

### PR DESCRIPTION
**Pull request by Developerayo**
https://deploy-preview-24--unstack.netlify.com/

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -24,14 +24,14 @@
...
-    <!-- Main Header-->
+     <!-- Main Header-->
     <header class="main-header">
         <!--Header-Upper-->
         <div class="header-upper">
             <div class="outer-container">
                 <div class="clearfix">
                     <div class="pull-left logo-box">
-                        <div class="logo"><a href="index.html"><img src="images/logo.png" alt="" title=""></a></div>
+                        <div class="logo"><a href="index.html"><img src="images/logo.png" alt="" title="" width="200px"></a></div>
```
Comment (kingisaac95) [inline about.html]: Let's have this in a style file.

File: about.html [hunk 2]
```diff
@@ -154,10 +154,10 @@ <h1>About Us</h1>
             <span class="icon icon-cross-1"></span>
             <span class="icon icon-circle-9"></span>
         </div>
-        <span class="float-text">About The</span>
+        <span class="float-text">About unSta</span>
```
Comment (kingisaac95) [inline about.html]:
```suggestion
        <span class="float-text">About unStack</span>
```

Comment (kingisaac95) [reply about.html]: The current floating text doesn't communicate the idea clearly. So let's have either of `About us` or `About unStack`

Comment (Developerayo) [reply about.html]: Done.

File: about.html (hunk 1 above)
Comment (Developerayo) [reply about.html]: Done.

**Comment by kingisaac95** [comment]
@Developerayo is this PR still valid or is it being replaced by the new one?

**Comment by Developerayo** [comment]
It's still valid @kingisaac95

**Pull request merged**